### DISCUSSION TITLE
[Projects Beta] Project board crashes when adding new column with specific names

### DISCUSSION THREAD

**Discussion by MikeBeas** [community/community#16874] [Projects and Issues] [answered]
On the new beta version of projects, attempting to add a column with a specific name crashes the board and displays a "The project failed to load" message. The column name in question is "Merged to Milestone" and the page crashes at "Merged to Milest" when the "t" key is pressed. (16th character causes crash.)

I thought this was possibly related to the length of the name, but "12345678901234567890" did not crash the page. However, "123456789 1234" did, when the "4" key is pressed. (14th character causes crash.)

"12345678901234567890 12345678901234567890" did not crash the page at all.

"This is a test colu" did when the "u" key is pressed. (19th character causes crash.)

Although a specific length does not seem to trigger the crash, the page does crash consistently on the same letter each time for each different phrase.

"Pull Request Ope" crashes on the "e" key. (16th character causes crash.)

The crash produces a minified React error which leads to this message: https://reactjs.org/docs/error-decoder.html/?invariant=185.

Not really sure what's going on here. The crashes seem to happen at consistent but random points in a lot of these strings. The stack trace doesn't look very useful since it's the minified version of everything but I can provide it if nobody is able to reproduce to problem.

(Also the reload button on the "the project failed to load" UI that appears upon crash isn't centered 😉) [edit: ahh it's centered today!]

**Comment by mattcosta7** [comment]
@MikeBeas we've been trying to repro this, and haven't been able to - can you confirm the browser, version, and any extensions that might be installed so we can try to recreate your environment a bit?

**Comment by MikeBeas** [comment]
@mattcosta7 Sure, I'm on Safari 15.5 (17613.2.7.1.8) for Mac (Montery 12.4). No browser extensions enabled. I just checked and I am **not** able to repro in Chrome v101. Seems like it could be a Safari issue.

**Comment by shiftkey** [comment, marked as answer]
> The crash produces a minified React error which leads to this message: https://reactjs.org/docs/error-decoder.html/?invariant=185.

Based on the notes and the fact that this seems to be related to editing the column title, there was a fix in yesterday's release to handling a similar problem - pasting text into the column title would also cause this `Maximum update depth exceeded.` issue.

I think we've incidentally addressed this issue by removing that problem component, so I'd be curious if you're able to reproduce it @MikeBeas. As I understand it, this isn't a browser-specific issue - just an unfortunate race condition.

**Comment by MikeBeas** [comment]
@shiftkey I can confirm the issue is resolved now, the page no longer crashes when entering the previously-problematic column titles!
